src/components/views/settings/tabs/room/RolesRoomSettingsTab.tsx

```diff
@@ -63,14 +63,14 @@ function parseIntWithDefault(val, def) {
     return isNaN(res) ? def : res;
 }
 
-export class BannedUser extends React.Component {
-    static propTypes = {
-        canUnban: PropTypes.bool,
-        member: PropTypes.object.isRequired, // js-sdk RoomMember
-        by: PropTypes.string.isRequired,
-        reason: PropTypes.string,
-    };
+interface IBannedUserProps {
+    canUnban: boolean;
+    member: RoomMember;
+    by: string;
+    reason: string;
```

Comment:
the PropType says this is optional, is this change intentional?

Reply:
No, will fix.